app/models/item.rb

```diff
@@ -6,4 +6,8 @@ class Item < ApplicationRecord
   validates :name, presence: true
   validates :description, presence: true
   validates :unit_price, presence: true, numericality: true
-end
\ No newline at end of file
+
+  def date_invoice_created
+    invoices.exists? ? invoices.first.format_date_created : "No Invoice for this Item"
```

Comment:
Looks like it's only running the formatting for an item's first invoice with the `.first`. Saw this issue was already noted in the PR description to revisit this problem later.